driver.go

```diff
@@ -88,10 +88,13 @@ func Register(driverName string, options ...TraceOption) (string, error) {
 
 // Wrap takes a SQL driver and wraps it with OpenCensus instrumentation.
 func Wrap(d driver.Driver, options ...TraceOption) driver.Driver {
-	o := TraceOptions{}
+	o := TraceOptions{
+		InstanceName: defaultInstanceName,
+	}
 	for _, option := range options {
 		option(&o)
 	}
+	o.DefaultAttributes = append(o.DefaultAttributes, trace.StringAttribute("sql.instance", o.InstanceName))
```

Comment:
Here is my primary concern... can you make it possible to configure if the `sql.instance` attribute is recorded or not. If people don't explicitly set there is hardly any value in repeating this attribute in each span.

Reply:
Would it be acceptable to add attribute if `o.InstanceName != ""`?

Reply:
@basvanbeek please check if it is better now.